app/models/room.rb

```diff
@@ -4,4 +4,10 @@ class Room < ApplicationRecord
   has_many :user_rooms, :dependent => :destroy
   has_many :users, through: :user_rooms
   has_secure_password
+
+  after_initialize :init
+
+  def init
+    self.game_started = false
+  end
```

Comment:
You can set up default values like this so you don't have to specify them every time you call `Room.create(...)`.